lib/bundix/source.rb

```diff
@@ -111,6 +111,7 @@ def format_hash(hash)
     end
 
     def fetch_local_hash(spec)
+      platform = Gem::Platform::RUBY # default platform value
```

Comment:
Ultimately, the issue seems to stem from this value remaining `nil` as never assigned for the gems that don't have platform dependent versions. This `nil` ended up being returned on https://github.com/inscapist/bundix/pull/12/files#diff-6c1d40dc95e017380bc6a0cd183b22f63fef5ac09356ae7ba8a288009a6355c7R128 .

I've amended it by pre-assigning the default value.